modules/microshift-ingress-controller-config.adoc

```diff
@@ -148,14 +270,20 @@ When configured, this field must contain a valid expression or the {microshift-s
 * When the policy is set to `Respond`, the ingress controller sends an HTTP `400` or `408` response, logs the connection if access logging is enabled, and counts the connection in the appropriate metrics.
 
 * When the policy is set to `Ignore`, the `http-ignore-probes` parameter is added to the `HAproxy` process configuration. After this parameter is added, the ingress controller closes the connection without sending a response, then either logs the connection or incrementing metrics.
-//Q: are the incrementing metrics in this case config'd by the admin, or are they automatic to our config?
+//Q: Does the admin need to configure Prometheus or Grafana to have incrementing metrics?
```

Comment:
No need to configure it in order to have the metrics incrementing, as that happens by itself, but yes to be able to see it.